extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigationWallNotifications_index.php

```diff
@@ -1,12 +1,12 @@
 <div id="notificationsEntryPoint" class="wds-global-navigation__notifications-menu wds-global-navigation__notifications-menu wds-dropdown notifications-container">
-	<div class="wds-dropdown__toggle wds-global-navigation__dropdown-toggle" title="<?=wfMessage( 'global-navigation-notifications-title' )->escaped()?>">
+	<div class="wds-dropdown__toggle wds-global-navigation__dropdown-toggle" title="<?=wfMessage( 'global-navigation-messages-title' )->escaped()?>">
```

Comment:
Are you sure about this? I thought the old notifications were supposed to have only the icon changed.